src/Controller/AuthenticationController.php

```diff
@@ -281,10 +269,10 @@ private function generateNotificationResponse(string $status): JsonResponse
 
     /**
      *
-     * @Route("/authentication/qr", name="app_identity_authentication_qr", methods={"GET"})
      * @throws InvalidArgumentException
      */
-    public function authenticationQrAction(): Response
+    #[Route(path: '/authentication/qr', name: 'app_identity_authentication_qr', methods: ['GET'])]
+    public function authenticationQr(): Response
     {
         $nameId = $this->authenticationService->getNameId();
         $sari = $this->stateHandler->getRequestId();
```

Comment:
sari? can this be more descriptive? (I know it is not a change in this PR)

Reply:
SARI is a domain term which should be known to step up developers. Its an authentication wide (from AR to Saml Response) request ID that is displayed on the error page. And helps us tremendously in grepping the log entries.